lib/inputstreamhelper/__init__.py

```diff
@@ -26,15 +26,14 @@ class InputStreamException(Exception):
 
 
 def cleanup_decorator(func):
-    """Decorator which runs cleanup before and after a function"""
+    """Decorator which runs cleanup after a function"""
 
-    def clean_before_after(self, *args, **kwargs):  # pylint: disable=missing-docstring
+    def clean_after(self, *args, **kwargs):  # pylint: disable=missing-docstring
         # pylint only complains about a missing docstring on py2.7?
-        self.cleanup()
```

Comment:
We had some rare situations previously, where ish would crash during install, which meant it didn't clean up. That wouldn't be a big issue by itself, but if the bin was partially extracted, or still mounted, when ish ran again, that could cause issues. See e.g. https://github.com/emilsvennesson/script.module.inputstreamhelper/issues/265#issuecomment-589849360 and the resulting PR #273. Also note that an unmount before/at the beginning of the install was already there before my change.

I think none of these issues could arise with the new ARM install method, so a cleanup before might not be necessary there, but while we still have the legacy ARM install method, I think this should stay. I was planning to remove the legacy method a few releases later, when the new one tried and tested. IMO the cleanup before can be removed then, too, since the only thing in temp will be the downloaded zips (the chrome os and the x86 for the license) and I think they would simply be overwritten, if they were already there.